data/sigma_config.yaml

```diff
@@ -20,6 +20,7 @@ logsources:
         - "bash:history:command"
         - "apt:history:line"
         - "selinux:line"
+        - "syslog"
```

Comment:
no you would use syslog:line if you want to specifically look into syslog logs. If you are looking for general linux executions, you look into datatype:syslog.

So e.g. this would include `syslog:ssh:login`. If you make it to accurate here to `syslog:line` you would miss out `syslog:ssh`. Hope that make sense